docs/index.rst

```diff
@@ -7,7 +7,7 @@ Welcome to python-zeroconf documentation!
 
 .. image:: https://img.shields.io/pypi/v/zeroconf.svg
     :target: https://pypi.python.org/pypi/zeroconf
-   
+
```

Comment:
Unrelated whitespace change, please restore this.

Reply:
Done, my ide removed it automatically on save